patches/react-native-reanimated+3.16.4+002+dontWhitelistTextProp.patch

```diff
@@ -1,5 +1,6 @@
 diff --git a/node_modules/react-native-reanimated/src/component/PerformanceMonitor.tsx b/node_modules/react-native-reanimated/src/component/PerformanceMonitor.tsx
 index 38e3d39..9936670 100644
+
```

Comment:
Do we need this newline?

Reply:
No, I just added it to have the reassure test instance see this as a file change and not just a rename but it didn't work. (also that part of the diff is ignored and is used as a comment section)